test: cleanup and add net48 on windows

viceice opened the pull request:
- remove obsolete msbuild properties
- test net6.0 and net48 (on windows)

zone117x reviewed (commented):
Thanks for the cleanup!

We need to implement github workflow CI (as opposed to the old and probably broken AppVeyor CI) that runs tests on a ubuntu, windows, and macos runners. Is this something you'd be able to do?

viceice commented:
sure, use this PR or a new one?

zone117x commented:
This PR makes sense

viceice commented:
Sample run: 
- https://github.com/visualon/MimeMapping/actions/runs/3470971015

zone117x commented on .github/workflows/build.yml:
```diff
@@ -0,0 +1,88 @@
+        with:
+          fetch-depth: 2
+
+      - uses: actions/cache@9b0c1fce7a93df8e3bb8926b0d6e9d89e92f20a7 # v3.0.11
```
For this action and the others, thank you!

viceice replied:
sure, but would like to use renovate to automatically update those.

using git Sha is much more secure because they are immutable.

viceice replied:
done

zone117x commented:
> you need to allow or approve the actions

For some reason I'm not using the usual Github UI for this. I've checked the repo permissions and everything looks good. Not sure what's going on there. Thanks for the example results -- I think we can just merge this anyway. 

Are you interested in becoming a maintainer for this repo?

viceice commented:
> > you need to allow or approve the actions
> 
> For some reason I'm not seeing the usual Github UI for this. I've checked the repo permissions and everything looks good. Not sure what's going on there. Thanks for the example results -- I think we can just merge this anyway. 

ok


> 
> Are you interested in becoming a maintainer for this repo?

sure, would like to automate a lot of things then
- updates via renovate bot
- semiautomatic releases via GitHub actions

so we can easily release new versions .

maybe we can also automate releases fully later.

zone117x commented:
@viceice that sounds good to me. I added you as a collaborator. I've also added a `NUGET_API_KEY` repo secret for this nuget package, so automated releases should be possible. Do you want to try opening this PR as a branch to see if that will trigger the new github workflow you've added?

viceice commented:
ok, thanks. will try to find some time tomorrow. (TZ=Berlin) 🤗

viceice commented:
> @viceice that sounds good to me. I added you as a collaborator. I've also added a `NUGET_API_KEY` repo secret for this nuget package, so automated releases should be possible. Appveyor has also been removed from CI. Do you want to try opening this PR as a branch to see if that will trigger the new github workflow you've added?

✅ 


closing in favor of #22
